pyls/plugins/jedi_completion.py

```diff
@@ -89,7 +89,10 @@ def _label(definition):
 
 
 def _detail(definition):
-    return definition.parent().full_name or ''
+    try:
+        return definition.parent().full_name or ''
+    except AttributeError:
+        return ''
```

Comment:
I think this needs to be changed to

    return definition.full_name or ''

for Jedi 0.14, at least according to the tests in

https://github.com/davidhalter/jedi/blob/master/test/test_api/test_full_name.py

Reply:
The change you made was not my idea to fix this. I think to cover both Jedi 0.13 and 0.14, this should be

```python
try:
    return definition.parent().full_name or ''
except AttributeError:
    return definition.full_name or ''
```